resource/writers/match_writers.cpp

```diff
@@ -453,6 +508,11 @@ json_t *jgf_match_writers_t::emit_vtx_base (const f_resource_graph_t &g,
     return o;
 }
 
+std::string jgf_match_writers_t::get_node ()
+{
+    return nodename;
```

Comment:
Fluxion designates member variables with `m_`:
```suggestion
    return m_nodename;
```